packages/opentelemetry-exporter-collector/src/platform/node/util.ts

```diff
@@ -54,19 +55,18 @@ export function sendWithHttp<ExportItem, ServiceRequest>(
       collector.logger.debug(`statusCode: ${res.statusCode}`);
       onSuccess();
     } else {
-      collector.logger.error(`statusCode: ${res.statusCode}`);
-      onError({
-        code: res.statusCode,
-        message: res.statusMessage,
-      });
+      const error = new collectorTypes.CollectorExporterError(
+        res.statusMessage,
+        res.statusCode
+      );
+      globalErrorHandler(error);
+      onError(error);
     }
   });
 
   req.on('error', (error: Error) => {
-    collector.logger.error('error', error.message);
-    onError({
-      message: error.message,
-    });
+    globalErrorHandler(error);
+    onError(error);
```

Comment:
well i believe it's the case here too, `globalErrorHandler` should only be called inside the `CollectorExporterBase` i think

Reply:
You're right, thanks again! I made another pass through and cleaned redundant calls to the `globalErrorHandler`.